pkgs/top-level/all-packages.nix

```
@@ -9428,7 +9428,8 @@ with pkgs;
 
   innoextract = callPackage ../tools/archivers/innoextract { };
 
-  input-remapper = python3Packages.callPackage ../tools/inputmethods/input-remapper { };
+  #python310 as pydbus does not build on 3.11
+  input-remapper = python310Packages.callPackage ../tools/inputmethods/input-remapper { };
```

pydbus supports python3.11 as of #263669
```suggestion
  input-remapper = python3Packages.callPackage ../tools/inputmethods/input-remapper { };
```

---

oh nice

---

reverting that might have made the nixos test hang or this laptop i tested on is just way too slow, asked ofborg to run them

---

looks like it passed on ofborg